### PR TITLE
Updates to accommodate the latest container changes.

### PR DESCRIPTION
**Pull request by geeknoid**

### CODE REVIEW

File: files/Makefile [hunk 1]
```diff
@@ -36,28 +36,24 @@ else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 5),armv8)
 else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 4),armv)
     TARGET_ARCH ?= arm
 else
-   $(error "This system's architecture $(LOCAL_ARCH) isn't recognized/supported")
+    $(error "This system's architecture $(LOCAL_ARCH) isn't recognized/supported")
```
Comment (sdake) [inline files/Makefile]: errors should not use quotes.  otherwise the quotes are printed...

File: files/Makefile [hunk 2]
```diff
@@ -36,28 +36,24 @@ else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 5),armv8)
...
+    TARGET_OS ?= linux
+    READLINK_FLAGS="-f"
 else ifeq ($(LOCAL_OS),Darwin)
-   TARGET_OS ?= darwin
-   READLINK_FLAGS=""
+    TARGET_OS ?= darwin
+    READLINK_FLAGS=""
 else
-   $(error "This system's OS $(LOCAL_OS) isn't recognized/supported")
+    $(error "This system's OS $(LOCAL_OS) isn't recognized/supported")
```
Comment (sdake) [inline files/Makefile]: errors should not use quotes.  error prints anything - including quotes.

Comment (geeknoid) [reply files/Makefile]: Fixed

File: files/Makefile [hunk 3]
```diff
@@ -36,28 +36,26 @@ else ifeq ($(shell echo $(LOCAL_ARCH) | head -c 5),armv8)
...
+    READLINK_FLAGS=""
 else
-   $(error "This system's OS $(LOCAL_OS) isn't recognized/supported")
+    $(error This system's OS $(LOCAL_OS) isn't supported)
 endif
 
-REPO_ROOT = $(shell git rev-parse --show-toplevel)
-REPO_NAME = $(shell basename $(REPO_ROOT))
-TARGET_OUT ?= $(HOME)/istio_out/$(REPO_NAME)
+export TARGET_OUT ?= $(shell pwd)/out/$(TARGET_ARCH)_$(TARGET_OS)
```
Comment (sdake) [inline files/Makefile]: clever that this ends up as /work.  Hopefully people remember not to change the work path ;)

**Pull request merged**